modules/proxy.js

```diff
@@ -1,62 +1,273 @@
-module.exports = async ({msg,bot,members,cfg}) => {
-    if(msg.channel.guild && (!msg.channel.permissionsOf(bot.user.id).has("readMessages") || !msg.channel.permissionsOf(bot.user.id).has("sendMessages"))) return;
-	if(members[0] && !(msg.channel.type == 1)) {
-		let clean = msg.cleanContent || msg.content;
-		clean = clean.replace(/(<a?:.+?:\d+?>)|(<@!?\d+?>)/,"cleaned");
-		let cleanarr = clean.split("\n");
-		let lines = msg.content.split("\n");
-		let replace = [];
-		let current = null;
-		for(let i = 0; i < lines.length; i++) {
-			let found = false;
-			members.forEach(t => {
-				let res = bot.checkMember(msg, t, cleanarr[i]);
-				if(res >= 0) {
-					if(t.brackets[res*2+1].length == 0) current = t;
-					else current = null;
-					found = true;
-					replace.push([msg,cfg,t,t.show_brackets ? lines[i] : lines[i].substring(t.brackets[res*2].length, lines[i].length-t.brackets[res*2+1].length)]);
-				}
-			});
-			if(!found && current) 
-				replace[replace.length-1][3] += "\n"+lines[i];
+const request = require("got");
+const strlen = require("string-length");
+const { crypto_stream_PRIMITIVE } = require("sodium-native");
```

Comment:
what's this for?

Reply:
Uh no idea, I'm going to blame intellisense and mashing keys on that one.

(sodium-native package was added by you [here](https://github.com/Keterr/Tupperbox/commit/3c6fb617993fdfcff6204586145d441fe757fbcc#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R25))